doc/060_forget.rst

```diff
@@ -312,11 +318,12 @@ four Sundays, but remove the rest:
    ---------------------------------------------------------------
    8 snapshots
 
-The result of the ``forget --keep-daily`` operation does not depend on when it
-is run, it will only count the days for which a snapshot exists. This is a
-safety feature: it prevents restic from removing snapshots when no new ones are
-created. Otherwise, running ``forget --keep-daily 4`` on a Friday (without any
-snapshot Monday to Thursday) would remove all snapshots!
+The result of the ``forget --keep-daily`` operation only partially depends on
+when it is run; it will only count the days for which a snapshot exists,
+although snapshots with a `time` lying in the future are ignored and never
```

Comment:
Future timestamps are only ignored by the `--keep-within*` options. And the addition of the remark here causes the next sentence "This is a safety feature" to be unclear what it refers to.

Reply:
Upon re-reading the paragraph, the future timestamp thing indeed seems kind of like a stray remark in between two connected sentences. That futures are ignored for `keep-within` is already mentioned in a `.. note::` section above. I've just removed the sentence in my upcoming commit but just treat that as a proposal.